content/docs/command-reference/add.md

```diff
@@ -6,16 +6,26 @@ file.
 ## Synopsis
 
 ```usage
-usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
-               [--glob] [--file <filename>] [-o <path>]
-               [--to-remote] [-r <name>] [-j <number>] [-f]
+usage: dvc add [-h] [-q | -v] [-f] [--no-commit] [--external]
+               [--glob] [-o <path>]
+               [--to-remote] [-r <name>] [--remote-jobs <number>]
                [--desc <text>] [--meta key=value] [--label <str>]
                [--type <str>]
```

Comment:
```suggestion
```

These options were also removed in 3.0. @dberenbaum, do you mind removing these in the same PR?

Reply:
Added, PTAL